presto-main/src/main/java/com/facebook/presto/server/security/PasswordAuthenticatorManager.java

```diff
@@ -60,11 +60,12 @@ public void loadPasswordAuthenticator()
             return;
         }
 
-        Map<String, String> properties = new HashMap<>(loadProperties(CONFIG_FILE));
+        File configFileLocation = CONFIG_FILE.getAbsoluteFile();
+        Map<String, String> properties = new HashMap<>(loadProperties(configFileLocation));
```

Comment:
maybe it is `com.facebook.presto.util.PropertiesUtil#loadProperties` which should be fixed? What do you think?

Reply:
Fair idea. Anyway, we take absolute path here for additional error reporting, so this would be _additionally_ not _instead_.